x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -511,10 +514,15 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
       />
     );
   }, [kibanaDataViews]);
+
   const DataSource = useMemo(() => {
     return (
-      <RuleTypeEuiFormRow label={i18n.SOURCE} $isVisible={true} fullWidth>
-        <EuiFlexGroup direction="column" gutterSize="s">
+      <RuleTypeEuiFormRow id="dataSourceSelector" label={i18n.SOURCE} $isVisible={true} fullWidth>
```

Comment:
Instead of wrapping this element with `<EuiTourStep />`, we can associate them via element id. When defining a step, we can specify this id in the `anchor` property.